src/main/java/uk/gov/beis/els/util/StringUtils.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.beis.els.util;
+
+public class StringUtils {
+
+  private static final int DECIMAL_PLACES_DEFAULT = 0;
+
+  public static String toPercentage(float number) {
+    return toPercentage(number, DECIMAL_PLACES_DEFAULT);
```

Comment:
Nit would just call `toPercentage(number, 0);` here, its clear what its doing. Don't think it needs a static var